lib/src/atoms/file/unarchive.rs

```diff
@@ -0,0 +1,46 @@
+use std::{fs::File, path::PathBuf};
+
+use flate2::read::GzDecoder;
+use tar::Archive;
+
+use crate::atoms::Atom;
+
+use super::FileAtom;
+
+pub struct Unarchive {
+    pub origin: PathBuf,
+    pub dest: PathBuf,
+}
+
+impl FileAtom for Unarchive {
+    fn get_path(&self) -> &PathBuf {
+        &self.origin
+    }
+}
+
+impl Atom for Unarchive {
+    // Determine if this atom needs to run
+    fn plan(&self) -> bool {
+        self.origin.exists()
```

Comment:
What happens, when `dest` already exists? A `force: <true|false>` could be solution. If `force` doesn't exist and `dest` exists, we should hinder the atom to run.